pod/perldelta.pod

```diff
@@ -5,35 +5,216 @@
+
+These are inspired by the same-named functions in the L<List::Util> module,
+except that they are implemented as direct core operators, and thus perform
+faster, and do not produce an additional subroutine call stack frame for
+invoking the code block.
+
+The feature flags enabling those keywords have been named L<keyword_any>
+and L<keyword_all> to avoid confusion with to the ability of the
+C<feature> module to refer to all of its features by using the C<:all>
+export tag.  [L<GH #23104|https://github.com/Perl/perl5/issues/23104>]
+
+The related experimental warning flags are consequently named
+C<experimental::keyword_any> and C<experimental::keyword_all>.
+
+=head2 Apostrophe as a global name separator can be disabled
+
+This was deprecated in Perl 5.38 and removed as scheduled in perl
+5.41.3, but after some discussion has been reinstated by default.
+
+This can be controlled with the C<apostrophe_as_package_separator>
+feature which is enabled by default, but is disabled from the 5.41
+feature bundle onwards.
+
+If you want to disable use within your own code you can explicitly
+disable the feature:
+
+  no feature "apostrophe_as_package_separator";
+
+Note that disabling this feature only prevents use of apostrophe as a
+package separator within code; symbolic references still treat C<'> as
+C<::> with the feature disabled:
+
+  my $symref = "My'Module'Var";
+  # default features
+  my $x = $My'Module'Var; # fine
+  no feature "apostrophe_as_package_separator";
+  no strict "refs";
+  my $y = $$symref;       # like $My::Module::Var
+  my $z = $My'Module'Var; # syntax error
+
+[L<GH #22644|https://github.com/Perl/perl5/issues/22644>]
 
-[ List each enhancement as a =head2 entry ]
+=head2 Lexical method declaration using C<my method>
+
+Like C<sub> since Perl version 5.18, C<method> can now be prefixed with the
+C<my> keyword.  This declares a subroutine that has lexical, rather than
+package visibility.  See L<perlclass> for more detail.
+
+=head2 Lexical method invocation operator C<< ->& >>
+
+Along with the ability to declare methods lexically, this release also permits
+invoking a lexical subroutine as if it were a method, bypassing the usual
+name-based method resolution by name.
+
+Combined with lexical method declaration, these two new abilities create the
+effect of having private methods.
+
+=head2 Switch and Smart Match operator reinstated
+
+The "switch" feature and the smartmatch operator, C<~~>, were introduced in
+v5.10.  Their behavior was significantly changed in v5.10.1.  When the
+"experiment" system was added in v5.18.0, switch and smartmatch were
+retroactively declared experimental.  Over the years, proposals to fix or
+supplement the features have come and gone.
+
+They were deprecated in Perl v5.38.0 and scheduled for removal in Perl
+5.42.0, and entirely removed in Perl 5.41.3.
+
+After some discussion these have been re-instated in 5.41.9.
+
+Using them no longer produces a deprecation warning.
```

Comment:
Probably too much detail for this perldelta. How about something along the lines of:

"They were deprecated in Perl v5.38.0 and scheduled for removal in Perl 5.42.0, but after some discussion their removal has been indefinitely postponed. Using them no longer produces a deprecation warning."